lib/main.js

```diff
@@ -131,6 +131,21 @@ export default {
         },
       }),
     );
+
+    this.subscriptions.add(
+      atom.workspace.observeTextEditors((editor) => {
+        editor.onDidSave(() => {
+          const validScope = editor.getCursors().some(cursor =>
+            cursor.getScopeDescriptor().getScopesArray().some(scope =>
+              grammarScopes.includes(scope)));
+          if (validScope && atom.config.get('linter-tslint.fixOnSave')) {
```

Comment:
That should work, but for consistency with the code I recently added could you make sure it works with `textEditor.getRootScopeDescriptor().getScopesArray()`?